src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,120 @@
 package org.example;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
 public class Main {
+
+    static List<String> options = new ArrayList<>(List.of("add", "remove", "list", "exit"));
+    static List<List<String>> tasks = new ArrayList<>();
+    static Scanner scanner = new Scanner(System.in);
+
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        readTasks("tasks.csv");
+        selectOption();
+    }
+
+    public static void readTasks(String fileName) {
+        Path path = Paths.get(fileName);
```

Comment:
CLEAN-CODE: lokalni promenne v ramci metod pokud mozno vsude pouzivej nemenne (final), je to lepsi pro pripad ze by si omylem nekde zacal pak promennou menit.

**Prosim oprav tohle u vsech promennych kde to dava smysl.**

```suggestion
        final Path path = Paths.get(fileName);
```